inc/Integrations/Airtable/AirtableIntegration.php

```diff
@@ -30,26 +35,26 @@ public static function register_blocks_for_airtable_data_source(
 		$tables = $data_source->to_array()['service_config']['tables'];
 
 		foreach ( $tables as $table ) {
-			$query = self::get_query( $data_source, $table );
-			$list_query = self::get_list_query( $data_source, $table );
-
-			register_remote_data_block(
-				array_merge(
-					[
-						'title' => $data_source->get_display_name() . '/' . $table['name'],
-						'render_query' => [
-							'query' => $query,
-						],
-						'selection_queries' => [
-							[
-								'query' => $list_query,
-								'type' => 'list',
+				$query = self::get_query( $data_source, $table );
+				$list_query = self::get_list_query( $data_source, $table );
+
+				register_remote_data_block(
+					array_merge(
+						[
+							'title' => $data_source->get_display_name() . '/' . $table['name'],
+							'render_query' => [
+								'query' => $query,
+							],
+							'selection_queries' => [
+								[
+									'query' => $list_query,
+									'type' => 'list',
+								],
 							],
 						],
-					],
-					$block_overrides
-				)
-			);
+						$block_overrides
+					)
+				);
```

Comment:
I think the indent got off kilter here

Reply:
Yet `No fixable errors were found` when running `npm run lint:phpcs:fix` 😕

Reply:
🤷 I can look at it, still off though